### DISCUSSION TITLE
DME approaches

### DISCUSSION THREAD

**Discussion by simpilothr** [albar965/littlenavmap#1025] [Q&A]
I've been trying to implement a route where it includes a DME arc (in this case it's the KUDUL1B arrival at LDZD, link is: crocontrol.hr/UserDocsImages/AIS%20produkti/eAIP/2023-04-20-AIRAC/graphics/eAIP/5754121_LD_AD_2_LDZD_12_IAC_RWY04_VOR_en.pdf ), but so far I haven't managed to get it working.
The closest I got was by picking another arrival at the same airport where a "course to intercept" waypoint was present before the arc began, with other waypoints that were essentially radials at certain distances. I found the same type of waypoints for my arrival, I just can't seem to find a command to make it look like an arc.

Edit: and 17 minutes later I obviously found the arrival I needed, but I'm still curious. Regardless if it's an arrival or not, is it possible to implement a DME arc?

**Comment by albar965** [comment]
You have to select the right transition for the approach to get the DME arc.
![grafik](https://user-images.githubusercontent.com/15254956/236292215-e276575c-d71f-467a-b1af-4eadcddc0a6a.png)
Alex
